tutorials/2d/particle_systems_2d.rst

```diff
@@ -21,16 +21,18 @@ parameters and then adding randomness to them.
 Particle nodes
 ~~~~~~~~~~~~~~
 
-Godot provides two different nodes for 2D particles, :ref:`class_GPUParticles2D` and
-:ref:`class_CPUParticles2D`.
-GPUParticles2D is more advanced and uses the GPU to process particle effects, but that limits
-it to higher end graphics API, and in our case to the GLES3 renderer. For projects using
-the GLES2 backend, CPUParticles2D is a CPU-driven option with near feature parity with
-GPUParticles2D, but lesser performance. While GPUParticles2D is configured via a
-:ref:`class_ParticleProcessMaterial` (and optionally with a custom shader), the matching options
-are provided via node properties in CPUParticles2D (with the exception of the trail settings).
-You can convert a GPUParticles2D node into a CPUParticles2D node by clicking on the node in the
-inspector, and selecting "Convert to CPUParticles2D" in the "Particles" menu of the toolbar.
+Godot provides two different nodes for 2D particles, :ref:`class_GPUParticles2D`
+and :ref:`class_CPUParticles2D`. GPUParticles2D is more advanced and uses the
+GPU to process particle effects, but that limits it to higher end graphics API,
+and in our case to the GLES3 renderer. CPUParticles2D is a CPU-driven option
```

Comment:
We should remove the reference to the GLES3 renderer here. Instead we should include a line saying that CPUParticles may be preferable on low-end systems or games with GPU bottlenecks.